example-mlem-get-started/code/README.md

```diff
@@ -0,0 +1,44 @@
+# MLEM Get Started
+
+This is an auto-generated repository for use in MLEM
+[Get Started](https://mlem.ai/doc/get-started). It is a step-by-step quick
+introduction into basic MLEM concepts.
+
+- The `main` branch contains only the code needed to start the Get Started. This
+  branch is useful if you want to go over Get Started executing all the commands
+  by yourself.
+- The `simple` branch contains the results of following Get Started. Results of
+  each step can be found in git tags.
+- The `dvc` branch contains the results of the
+  [MLEM+DVC tutorial](http://mlem.ai/doc/use-cases/dvc).
+
+🐛 Please report any issues found in this project here -
+[example-repos-dev](https://github.com/iterative/example-repos-dev).
+
+## Installation
+
+Python 3.7+ is required to run code from this repo.
+
+To be able to commit and push your results please fork and then clone this repo
+
+```console
+$ gh repo fork https://github.com/iterative/example-mlem-get-started
+$ cd example-mlem-get-started
+```
+
+Now let's install the requirements. But before we do that, we **strongly**
+recommend creating a virtual environment with a tool such as
+[virtualenv](https://virtualenv.pypa.io/en/stable/):
```

Comment:
we mention it, but we don't use (instead we use the Python's built-in venv module). As we agreed it's fine, but we should remove this link then